hs/src/Reach/CommandLine.hs

```diff
@@ -34,6 +36,34 @@ data CompilerOpts = CompilerOpts
   , co_verifyFirstFailQuit :: Bool
   }
 
+newtype SupportToolArgs = SupportToolArgs {sta_so :: SupportOpts}
+
+data SupportOpts = SupportOpts
+  { so_fileToUpload1 :: FilePath
+  , so_fileToUpload2 :: FilePath
+  , so_fileToUpload3 :: FilePath
+  , so_fileToUpload4 :: FilePath
+  , so_fileToUpload5 :: FilePath
+  , so_fileToUpload6 :: FilePath
+  , so_fileToUpload7 :: FilePath
+  , so_fileToUpload8 :: FilePath
+  , so_fileToUpload9 :: FilePath
+  }
+
+supportFromCommandLineHs :: Parser SupportToolArgs
+supportFromCommandLineHs =
+  SupportToolArgs
+    <$> (SupportOpts
+           <$> strArgument (metavar "path/to/file1" <> value "")
+           <*> strArgument (metavar "path/to/file2" <> value "")
+           <*> strArgument (metavar "path/to/file3" <> value "")
+           <*> strArgument (metavar "path/to/file4" <> value "")
+           <*> strArgument (metavar "path/to/file5" <> value "")
+           <*> strArgument (metavar "path/to/file6" <> value "")
+           <*> strArgument (metavar "path/to/file7" <> value "")
+           <*> strArgument (metavar "path/to/file8" <> value "")
+           <*> strArgument (metavar "path/to/file9" <> value ""))
```

Comment:
This is weird... why not look at the list of the arguments?

Reply:
I did it this way because it was easier for me to follow the example of [how the compiler does it](https://github.com/reach-sh/reach-lang/blob/84664bbc98d22b89ff491d7106b46caabaa727bb/hs/src/Reach/CommandLine.hs#L50).

Reply:
I did it this way because it was easier for me to follow the example of [how the compiler does it](https://github.com/reach-sh/reach-lang/blob/84664bbc98d22b89ff491d7106b46caabaa727bb/hs/src/Reach/CommandLine.hs#L50).

Reply:
I believe [it does it that way, now](https://github.com/reach-sh/reach-lang/pull/1063/files#diff-a164f5d393c75753025715ef5a521eb858cb0788f0782f179998322f24d730f9R49).